src/controllers/firePreventionPlan.controller.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
```

Comment:
This looks like it's a merge issue as the firePreventionPlan controller is now inside the upload folder.

Reply:
Deleted file as it is now in a different place (was a merge problem).